### PR TITLE
added rest of entities to complete the database

### PR DESCRIPTION
**Pull request by BigmanRecords**
a big commit again but will be smaller from now on as entity models are setup and will be only features.

### CODE REVIEW

File: src/main/java/com/bfwg/model/Review.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    @Column(name = "rating")
+    private Double rating;
+
+    @Column(name = "comment")
+    private String comment;
+
+    @NotNull
+    @Column(name = "entity")
+    private String entity;
```
Comment (StefanPahlplatz) [inline src/main/java/com/bfwg/model/Review.java]: Why is entity a string? A foreign key to vehicle makes more sense I think.

File: src/main/java/com/bfwg/model/Review.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package com.bfwg.model;
+
+import javax.annotation.sql.DataSourceDefinition;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "REVIEW")
+public class Review implements Serializable {
```
Comment (StefanPahlplatz) [inline src/main/java/com/bfwg/model/Review.java]: How do we know which user posted a review?

File: src/main/java/com/bfwg/model/Review.java (hunk 1 above)
Comment (BigmanRecords) [reply src/main/java/com/bfwg/model/Review.java]: can change the name to type as it is the string which says if the review is about a vehicle or user. could also be a boolean like isuser or isvehicle.

File: src/main/java/com/bfwg/model/Review.java (hunk 2 above)
Comment (BigmanRecords) [reply src/main/java/com/bfwg/model/Review.java]: will be the vehicle owner from reservation or the user from reservation depending on type/entity but i see your point might be better to add a field for the review writer

File: src/main/java/com/bfwg/model/Review.java (hunk 1 above)
Comment (BigmanRecords) [reply src/main/java/com/bfwg/model/Review.java]: changed name to type

File: src/main/java/com/bfwg/model/Review.java (hunk 2 above)
Comment (BigmanRecords) [reply src/main/java/com/bfwg/model/Review.java]: added user to review to see who made the review

**Pull request merged**